packages/component-library/src/components/plmg-avatar/plmg-avatar.scss

```diff
@@ -0,0 +1,68 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Common styles to all shadow-DOM components
+// Remove this section if your component does not use shadow DOM
+// -------------------------------------------------
+:host {
+  display: block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+// ------------- END Common styles -----------------
+
+// Component style
+// -------------------------------------------------
+
+.plmg-avatar {
+  background-color: tokens.$plmg-color-background-neutral-medium;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: tokens.$plmg-color-icon-neutral;
```

Comment:
The spec says:
> The background color, icon color is not fixed and can be modified

Please update.

Reply:
So two extra props: `iconColor` and `backgroundColor` (if not passed, colors set to the defaults)?